docs/pages/style-spec.js

```diff
@@ -254,7 +254,7 @@ export default class extends React.Component {
                         </ul>
                         <p>Developers using the <a href='https://www.mapbox.com/ios-sdk/'>Mapbox Maps SDK for iOS</a> or <a
                             href='https://github.com/mapbox/mapbox-gl-native/tree/master/platform/macos/'>
```

Comment:
I didn’t realize it, but this link is a bit outdated. The macOS map SDK has [a homepage](https://mapbox.github.io/mapbox-gl-native/macos/) now, and it doubles as the SDK’s API reference. The iOS and macOS API references are pretty similar, but some details are tailored to one platform versus the other.